src/components/sortOption/sortOption.tsx

```diff
@@ -0,0 +1,58 @@
+import { createMaterialTopTabNavigator } from '@react-navigation/material-top-tabs';
+import React, { useState } from 'react';
+import ProductsContainer from '../product/ProductsContainer';
+import { Platform, SafeAreaView, ScrollView, Text, TouchableOpacity, View } from 'react-native';
+
+interface Props {
+    data: object
+    label: object[]
+}
+
+const SortOption: React.FC<Props> = (props) => {
+    const { data, label } = props
+    const [renderData, setRenderData] = useState(data)
+    const [status, setStatus] = useState('Mới nhất')
+    const setStatusFilter = (item: string) => {
+        setStatus(item)
+    }
+
+    const getNewProduct = () => {
+        if (status === 'Mới nhất') {
+            let newProduct: [] = []
+            Array.isArray(data)
+                ? data.map(item => item.retailerTotalQuantity > 0 ? newProduct.push(item) : '') : ''
+
```

Comment:
Toán tử 3 ngôi (? : ) chỉ nên dùng khi cần trả về giá trị
ví dụ  
```js
let b = 1, c =2;
let a = c > b ? "asd" : "123"
```
những trường hợp này nên dùng if để check điều kiện

```suggestion
            if(Array.isArray(data))
                data.map(item => item.retailerTotalQuantity > 0 ? newProduct.push(item) : '')

```